pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -497,9 +497,17 @@ def plot_field(
             ``"CutPlane"``, ``"Surface"``, and ``"Volume"``.
         setup : str, optional
             Setup and sweep name on which create the field plot. Default is None for nominal setup usage.
-        intrinsics : dict, optional.
-            Intrinsic dictionary that is needed for the export.
-            The default is ``None`` which try to retrieve intrinsics from setup.
+        intrinsics : dict, str, optional
+            This parameter represents the value of the intrinsics variable of the simulation
```

Comment:
```suggestion
            Simulation intrinsic variables required to compute the field before the export. These are typically: frequency, time and phase.
```

@maxcapodi78 I can't select everything I want but it's from "This parameter ..." till "...before the export".